### DISCUSSION TITLE
Hackathon 2/3 November 2023

### DISCUSSION THREAD

**Discussion by gareth-j** [4DModeller/fdmr#231] [Announcements]
Welcome to our 2023 Hackthon!

## Getting help

If you need help using the `fdmr` package please open a [discussion here](https://github.com/4DModeller/fdmr/discussions/categories/q-a).

## Opening an issue

If you think you've found a bug in `fdmr` or want to request a new feature please [open an issue](https://github.com/4DModeller/fdmr/issues/new/choose).

**Comment by mnky9800n** [comment]
![image](https://github.com/4DModeller/fdmr/assets/942355/ba008667-b1c5-4301-b420-f9ab40e8b554)
